tabular/src/autogluon/tabular/task/tabular_prediction/tabular_prediction.py

```diff
@@ -432,6 +429,9 @@ def fit(train_data,
                 List of IP addresses corresponding to remote workers, in order to leverage distributed computation.
             visualizer : str, default = None
                 How to visualize the neural network training progress during `fit()`. Options: ['mxboard', 'tensorboard', None].
+            unlabeled_data : pd.DataFrame, default = None
+            Collection of data without labels that we can use to pretrain on. This is the same schema as X, but
```

Comment:
nit: indentation, what is X? Is this for pre-training specifically? Where is this used? Why would I use this?

Reply:
Added much more description about this arg.